draft-js-mention-plugin/src/mentionStrategy.js

```diff
@@ -1,12 +1,12 @@
 import { Entity } from 'draft-js';
 
-const findMention = (character) => {
+const findMention = (trigger) => (character) => {
   const entityKey = character.getEntity();
-  return (entityKey !== null && Entity.get(entityKey).getType() === 'mention');
+  return (entityKey !== null && Entity.get(entityKey).getType() === `mention-${trigger}`);
```

Comment:
This would be a breaking change to all existing implementations. I think by default the type should still be `mention`. I suggest we introduce another parameter to provide a custom `type`.